five-minute-serverless-backend/function.js

```diff
@@ -4,7 +4,7 @@ const CORS = (handler) =>
       statusCode: 200,
       headers: {
         'Access-Control-Allow-Origin': '*',
-        'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+        'Access-Control-Allow-Headers': context.request.headers["access-control-request-headers"],
```

Comment:
Nit: I think Access-Control-Allow-Headers is only needed when the (preflight OPTIONS) request includes `access-control-request-headers`, I think when the header is not sent (in the actual POST request) the response will be `Access-Control-Allow-Headers: undefined` which doesn't break anything but will be confusing for debugging.